packages/vite-plugin-lib/package.json

```diff
@@ -18,20 +18,18 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
       "require": "./dist/index.cjs",
       "import": "./dist/index.mjs"
     }
   },
   "main": "dist/index.cjs",
   "module": "dist/index.mjs",
-  "types": "dist/index.d.ts",
```

Comment:
I don't think `types` here is necessary, I suppose it couldn't hurt but in any modern system which vite would use, it will always go to `exports` anyways. I can add this back if you feel its important.

Reply:
I'd like to keep the `dist` type as-is. Did you experience any issues with the current output? `tsc`, `vue-tsc`, and VSCode all work fine with a single `d.ts` file. I know that's not the "correct" way, but it works just fine and has actual tool support.